tests/ApiCore/Tests/Unit/GapicClientTraitTest.php

```diff
@@ -386,6 +386,7 @@ public function buildClientOptionsProvider()
             'serviceName' => 'test.interface.v1.api',
             'clientConfig' => __DIR__ . '/testdata/test_service_client_config.json',
             'descriptorsConfigPath' => __DIR__.'/testdata/test_service_descriptor_config.php',
+            'gcpApiConfigPath' => __DIR__.'/../testdata/test_service_grpc_config.json',
```

Comment:
Maybe I am misunderstanding the change, but shouldn't we also need to add ['transportConfig']['grpc']['stubOpts'] to this?

Reply:
Done.